includes/sanitizers/class-amp-script-sanitizer.php

```diff
@@ -35,6 +35,11 @@ public function sanitize() {
 				continue;
 			}
 
+			// Skip noscript elements inside of amp-img. See \AMP_Img_Sanitizer::adjust_and_replace_node().
+			if ( 'amp-img' === $noscript->parentNode->nodeName ) {
```

Comment:
Actually, I think if the element name just starts with `amp-` we should skip. This will account for `noscript` being used as fallback for other elements, like `video`.